www/package.json

```diff
@@ -121,8 +104,5 @@
     "eslint": "8.19.0",
     "eslint-config-pluralsh": "3.1.0",
     "postcss-import": "8.2.0"
-  },
-  "peerDependencies": {
-    "react-transition-group": ">=4.4.2"
```

Comment:
@dogmar it looks that you've added it but with the changes made here I've decided to put it back with all dependencies. It looks like we are using it directly in the code. Please let me know if you would rather not change it.